config/main.py

```diff
@@ -2290,6 +2360,56 @@ def del_vrf(ctx, vrf_name):
         config_db.set_entry('VRF', vrf_name, None)
 
 
+@vrf.command('add_vrf_vni_map')
+@click.argument('vrfname', metavar='<vrf-name>', required=True, type=str)
+@click.argument('vni', metavar='<vni>', required=True)
+@click.pass_context
+def add_vrf_vni_map(ctx, vrfname, vni):
+    config_db = ctx.obj['config_db']
+    found = 0
+    if vrfname not in config_db.get_table('VRF').keys():
+        ctx.fail("vrf {} doesnt exists".format(vrfname))
+    if not vni.isdigit():
+        ctx.fail("Invalid VNI {}. Only valid VNI is accepted".format(vni))
+
+    if (int(vni) < 1) or (int(vni) > 16777215):
```

Comment:
Remove all hardcoded values and convert them into constants or descriptive names.

Reply:
Called a routine to validate the vnid value.